lib/addons/cert-manager/index.ts

```diff
@@ -0,0 +1,82 @@
+// lib/certmanager_addon.ts
+import { Construct } from 'constructs';
+import merge from "ts-deepmerge";
+import { setPath } from '../../utils/object-utils';
+import { HelmAddOn, HelmAddOnUserProps, HelmAddOnProps } from "../helm-addon";
+import { ClusterInfo, Values } from "../../spi";
+import { createNamespace } from "../../utils";
+/**
+ * User provided options for the Helm Chart
+ */
+export interface CertManagerAddOnProps extends HelmAddOnUserProps {

```

Comment:
Do we need to expose prometheus options here? E.g. prometheus.enabled = false (true by default). 
@elamaran11 any issue on this add-on insalling prometheus?

Reply:
As you looking in to this page https://docs.aws.amazon.com/eks/latest/userguide/adot-reqts.html, the Cert Manager YAML uses the following annotations. So having prometheus enabled is required but we have to try it out once the Addon is in place.
```
      annotations:
        prometheus.io/path: "/metrics"
        prometheus.io/scrape: 'true'
        prometheus.io/port: '9402'
```

Reply:
@elamaran11 so there is no change required from cert-manager addon side right?

Reply:
I suppose we can go as is and adjust if this interferes with other add-ons. If prometheus configuration will become frequently, e.g. endpoint, port, etc. I would rather put those at the root level, than force users to understand the values.yaml format.

Reply:
@shapirov103 I had a word with @elamaran11 
I have verified that the above annotations are already present in my cert manager pods
metadata:
  annotations:
    kubernetes.io/psp: eks.privileged
    prometheus.io/path: /metrics
    prometheus.io/port: "9402"
    prometheus.io/scrape: "true"

So we agreed that we are good to go and no extra changes needs to be done here